pkg/reconciler/kubernetes/initcontroller/initcontroller.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2021 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package initcontroller
+
+import (
+	"context"
+
+	"github.com/go-logr/zapr"
+	mfc "github.com/manifestival/client-go-client"
+	mf "github.com/manifestival/manifestival"
+	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
+	"github.com/tektoncd/operator/pkg/reconciler/common"
+
+	"os"
+	"path/filepath"
+	"strings"
+
+	"go.uber.org/zap"
+	"knative.dev/pkg/injection"
+)
+
+type Controller struct {
+	Ctx              context.Context
+	Logger           *zap.SugaredLogger
+	VersionConfigMap string
+}
+
+type ResourceManifest struct {
+	Ctx              context.Context
+	Manifest         *mf.Manifest
+	VersionConfigMap string
+}
```

Comment:
```suggestion
type Controller struct {
	Ctx              context.Context
	Logger           *zap.SugaredLogger
	VersionConfigMap string
}
```

It looks like the sole purpose of Controlelr stuct is to act as a set of inputs InitController function can work on. In addition, this stuct is not used anywhere else. So let us avoid the additional ResourceManifest struct and make `fetchSourceManifests` a method of Controller struct.